routes/subscriptions.js

```diff
@@ -38,6 +38,9 @@ subscriptions.getAllSubscriptions = function (app, res, loggedInUserId, filter,
         if (!userInfo.admin && !userInfo.approver) {
             return utils.fail(res, 403, 'Not allowed. This is admin/approver land.');
         }
+        if(userInfo.approver) //add approver groups
+            filter['api_group']=userInfo.groups.join('|');
```

Comment:
Please keep the coding style; Alt+Shift+F in VS Code can help with that.

Reply:
See above - we need special handling here; we mustn't match for substrings (`% ... %`), but for exact matches of the `api_group`.

Reply:
Above, below,... whatever, see `pg-utils.js`: https://github.com/apim-haufe-io/wicked.api/pull/20#discussion_r262430850

Reply:
fixed